app/controllers/pages_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PagesController < ApplicationController
   def home
+    redirect_to articles_path if logged_in?
```

Comment:
und was passiert, wenn  ich nicht eingeloggt bin? Ist ein Template für `home` hinterlegt?

Reply:
Ja, in `views/layouts/pages/home.html.erb`
Wenn du nicht eingeloggt bist, dann bekommst du dieses Template :)

Reply:
ok